rust-code-analysis-book/src/languages.md

```diff
@@ -4,15 +4,12 @@ This is the list of programming languages parsed by
 **rust-code-analysis**.
 
 - [x] C++
-- [ ] C#
```

Comment:
Why not keeping C# and Go in the list?

Reply:
We do not support them right now

Reply:
We don't support Kotlin either, but you added it. I think it's good to list the ones we would like to support without the X

Reply:
We still don't have metrics for Kotlin but we support it as parsed language https://github.com/mozilla/rust-code-analysis/blob/master/Cargo.toml#L32